main.scss

```diff
@@ -20,3 +20,7 @@
 @import "src/scss/field-errors";
 @import "src/scss/field-valid";
 @import "src/scss/global-and-section-messages";
+
+// Set silent mode back on again to avoid it being output twice
+$o-forms-is-silent: true;
+$o-ft-forms-is-silent: true;
```

Comment:
I completely agree on the principle but unfortunately CSS optimisers don't de-dupe selectors that appear multiple times in the code, in order to preserve the cascade. 

As long as Sass has no "include once" directive, I'm not sure how we can do it another way than overriding the silent mode right after the module was outputted.